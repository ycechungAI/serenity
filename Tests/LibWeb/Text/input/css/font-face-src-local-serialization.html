<script src="../include.js"></script>
<script>
    test(() => {
        let style = document.createElement("style");
        style.innerText = "@font-face { font-family: 'a1'; src: local('xyz'); }";
        document.head.appendChild(style);
        let sheet = style.sheet;
        println(sheet.cssRules[0].cssText);
    });
</script>
