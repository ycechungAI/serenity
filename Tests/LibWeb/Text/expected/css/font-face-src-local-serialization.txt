@font-face { font-family: "a1"; src: local("xyz"); unicode-range: "U+0-10ffff"; }
